lib/std/json.zig

```diff
@@ -1105,6 +1105,10 @@ pub const TokenStream = struct {
         };
     }
 
+    fn stackUsed(self: *TokenStream) u8 {
+        return self.parser.stack_used + if (self.token != null) @as(u8, 1) else 0;
```

Comment:
I wonder if this overflows when the innermost token in a 256-deep-stack is a number?
If so, will need to upgrade the return value to a `u9` (and do some casts)

Reply:
Tried with `skipValue`, but it didn't panic on an overflow. Should I add a test specific to TokenStream too?